app/views/user_mailer/collection_notification.text.erb

```diff
@@ -1,10 +1,8 @@
 <% content_for :message do %>
-Dear Collection Maintainers!
-
 You have received a message about your collection "<%= @collection.title %>" (<%= collection_url(@collection) %>):
 <%= text_divider %>
 
 <%= raw @message %>"
```

Comment:
*twitches* I just noticed this stray quotation mark.

Reply:
I won't mind if you deal with it here. We don't even need manual testing for it.

Reply:
Whoops, forgot about that. Done now!